src/test/data/ManualTesting/testScript.md

```diff
@@ -0,0 +1,233 @@
+* `add <name> by <deadline>` - adds a task which have to be done by the specified deadline. Note the keyword `by`.
+* `add <name> from <start time> to <end time>` - adds a event which will take place between start time and end time. Note the keyword `from` and `to`.
+
+
+### 1. Add a floating task
+Type `add watch movie`.
+#### Result 
+A new floating task named "watch movie" is created at the top of **Do It Anytime** column, and highlighted in purple borders.
+
+### 2. Add a task with deadline
+Type `add submit essay by 10pm`.
+#### Result 
+A new task named "submit essay" is created in the **Do It Soon** column, with its deadline under the name the this task. Also, it is highlighted in purple borders.
+
+### 3. Add a task with event time
+Type `add submit essay by 10pm`
+#### Result 
+A new task named "submit essay" is created in the **Do It Soon** column, with its deadline under the name the this task. Also, it is highlighted in purple borders.
+
+## Rename
+To rename a task, you have to start your command with the keyword `rename`.
+
+>Here is the *acceptable format*
+
+> * `rename <id> <new name>` - give a new name to the task identified by <id>. The <id> must be a positive number and be in the most recent to-do list displayed.
+
+Type `rename 1 complete peer review on TEAMMATES for 2101`.
+#### Result
+The task with index 1 is renamed to `complete peer review on TEAMMATES for 2101`. Also, it is highlighted in purple borders.
+
+## (Re)schedule 
+To reschedule a task, you have to start your command with the keyword `schedule`.
+
+>Here are the *acceptable format(s)*:
+
+>* `schedule <id>` - re-schedule the task identified by `<id>`. It can now be done anytime.  It is no longer bounded by a deadline or event time!
+* `schedule <id> by <deadline>` - set or update the deadline for the task identified. Note the keyword `by`.
+* `schedule <id> from <start time> to <end time>` - update the start/end time of the task identified by `<id>`. Note the keyword `from` and `to`.
+
+###1. Schedule a task with no deadline or event time
+Type `schedule 22`.
+####Result
+Task with index 22 now has no deadline, and it is shifted to **Do It Anytime** column with a new index. Also, it is highlighted in purple borders.
+
+###2. Schedule a task with a deadline
+Type `schedule 23 by 11 nov 10pm`.
+####Result
+Task with index 23 now has a deadline, and it is shifted to **Do It Soon** column with a new index. Also, it is highlighted in purple borders.
+
+###3. Schedule a task with event time
+Type `schedule 14 from 13 nov 2pm to 4pm`.
+####Result
+Task with index 14 now has a event time, and it is assigned with a new index. Also, it is highlighted in purple borders.
+
+## Mark
+To mark a task as completed, you have to start your command with the keyword `mark`.
+
+>Here is the *format*:
+
+>* `mark <id>...` - mark all the tasks identified by `<id>`(s) as completed. Each `<id>` must be a positive number and in the most recent to-do list displayed.
```

Comment:
I will update the main user guide. id are separated by commas or space and can also be inputed as a range